_docs-sources/pipelines/overview/configurations-as-code.md

```diff
@@ -254,6 +254,12 @@ In this example, the `deploy_branch_name` attribute is set to `main`, which mean
 *Supported Attributes:*
 
 - `deploy_branch_name` (Optional): The branch that Pipelines will deploy infrastructure changes from. If not set, Pipelines will deploy infrastructure changes from the `main` branch.
+- `consolidate_added_or_changed` (Optional): Whether or not Pipelines will consolidate added or changed resources when running Terragrunt commands. If not set, Pipelines will consolidate added or changed resources.
```

Comment:
I'm not sure users know what "consolidated" means, probably worth clarifying multiple jobs vs. `run-all`

Reply:
Thinking about it... this is a big topic, maybe we should link to a dedicated docs section about consolidation?

Reply:
Is this good enough for now?
https://github.com/gruntwork-io/docs/pull/1859/commits/0686f422ad272d5bb8fd77557627fbee44e91dd0

We can revisit later?